mne/preprocessing/xdawn.py

```diff
@@ -7,11 +7,12 @@
 
 import numpy as np
 from scipy import linalg
+from sklearn.preprocessing import LabelEncoder
```

Comment:
I believe it's this line that's causing the error. See https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_import_nesting.py#L29. This needs to be a nested import.